.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: scala
+scala:
+  - 2.11.7
```

Comment:
[did you mean: scala 2.12.2?](https://github.com/broadinstitute/sam/blob/develop/project/Settings.scala#L42)